draft-ietf-quic-http.md

```diff
@@ -1867,8 +1871,11 @@ FRAME_SIZE_ERROR (0x6):
 : HTTP_MALFORMED_FRAME error codes defined in {{http-error-codes}}.
 
 REFUSED_STREAM (0x7):
-: Not applicable, since QUIC handles stream management.  Would provoke a
-  STREAM_ID_ERROR from the QUIC layer.
+: When used to indicate that a request was not processed, HTTP_REQUEST_REJECTED
+  in {{http-error-codes}}.
+: When used to indicate that an active stream limit was exceeded, not applicable
```

Comment:
I had a little trouble parsing this bit.  Maybe reorder the sentences:

> HTTP_REQUEST_REJECTED (in {{http-error-codes}}) is used to indicate when requests were not processed.  Otherwise, not applicable because QUIC handles stream management.  A STREAM_ID_ERROR at the QUIC layer is used for streams that are improperly opened.